src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -591,9 +591,11 @@ public void visit(NodeTraversal t, Node n, Node parent) {
         }
       }
 
-      if (n.isName()) {
+      boolean isShorthandObjLitKey = n.isStringKey() && !n.hasChildren();
+      if (n.isName() || isShorthandObjLitKey) {
         String name = n.getString();
         if (suffix.equals(name)) {
+          // TODO(moz): This is highly unlikely, but do we really need to return early?
```

Comment:
Please phrase your TODO in the form of something to _do_ so that a future maintainer knows whether they need to take action or not.